src/evm_builtins.rs

```diff
@@ -99,14 +99,14 @@ impl Instructions for EVMInstructions {
             "calldatasize" => single_return(evm_context::calldatasize(ssa).into()),
             "calldatacopy" => panic!("Builtin {} not implemented", builtin.name), // TODO
             "codesize" => single_return(evm_context::codesize(ssa).into()),
-            "codecopy" => panic!("Builtin {} not implemented", builtin.name), // TODO
+            "codecopy" => vec![], //panic!("Builtin {} not implemented", builtin.name), // TODO
```

Comment:
Shouldn't this and `extcodecopy` havoc memory?

Reply:
Yes, this is just to prevent the system from crashing on my example.